src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Offender.kt

```diff
@@ -112,17 +120,22 @@ class OffenderBuilder(
   ) = bookingBuilderFactory.builder()
     .let { builder ->
       builder.build(
-        offender = offender,
-        // if you want multiple bookings then create them with latest booking first so it gets seq 1 like in Nomis
-        bookingSequence = bookingSequence ?: (offender.bookings.size + 1),
+        offender = aliasOffender,
+        // if you want multiple bookings then create them with latest booking first so it gets seq 1 like in Nomis,
+        // or override the sequences
+        bookingSequence = bookingSequence ?: nextBookingSequence++,
```

Comment:
I thought you'd removed the `bookingSequence` parameter from the test I added it to - so can we remove from the DSL entirely if it's not needed?

Reply:
Ah, well it turned out there is no other way to define a prisoner with the current booking on an alias and an old booking on the root. Without this, the first booking always gets bookingSequence = 1.

Reply:
Well you possibly could by declaring the alias before the old booking... but yes I never liked that the active booking has to be the first one declared, it feels counterintuitive.

We could try and be clever by holding onto the list of bookings and juggling the sequences around to make the active booking no.1, and also adding some validation to prevent more than 1 active booking.

But that sounds horrible so in complicated scenarios declaring the bookingSequence for each booking is far more explicit. So I think you're right and we should keep it.